web_src/js/components/RepoActionView.vue

```diff
@@ -121,6 +146,12 @@ const sfc = {
       currentJobStepsStates: [],
       artifacts: [],
       onHoverRerunIndex: -1,
+      menuVisible: false,
+      isFullScreen: false,
+      timeVisible: {
+        'stamp': false,
+        'duration': false,
```

Comment:
No need to use quotes?

```
        stamp: false,
        duration: false,
```

Or even two words (make these variable easier to search) ?

```suggestion
        showStamp: false,
        showDuration: false,
```

Reply:
Done in https://github.com/go-gitea/gitea/pull/24876/commits/47d38f1a53ec485f7ef869124d37856c436e29d3